code/modules/paperwork/paperbin.dm

```diff
@@ -61,11 +61,13 @@
 		if(!movable_atom.pixel_x)
 			movable_atom.pixel_x = rand(-3,3)
 	LAZYNULL(papers)
+	LAZYINITLIST(papers)
```

Comment:
this is silly. what youre doing here is setting the papers list to null and then initializing it as an empty list, but lazy lists being nulled when empty is actually a feature. 
the solution here is to make everything that adds or removes elements from the papers list into the lazylist macro version

Reply:
Looking at the various Lazylist stuff, is there a reason to not use `LAZYCLEARLIST` since when collapsing or burning, it removes every paper in the list. 
I'm just looking for ways that don't null the list since nulling prevents new papers from being added.

Reply:
the lazylist macros that add to the list automatically initialize the list if its null, and the lazy list macros that remove elements from the list automatically null the list if the last element is removed, its a feature of the macros that you can see in _lists.dm

the problem isnt that the list is null, its that its being treated like a lazylist in some places but not others. either way is fine really but you should make it consistently a lazylist or not a lazylist. if you want to keep it not a lazylist then replace both the LAZYNULL() and LAZYINITLIST() with papers.Cut()